migrations/20240702112229_migration_name.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+    
+export async function up(knex){
+    await knex.schema.createTable('games', function(table) {
+        table.bigIncrements('id').primary();
+        table.integer('number').notNullable();
```

Comment:
We name the column in a more self explanatory name. You can use `checkBetween` to specify the range of the values the column can take.